pbsmetrics/go_metrics.go

```diff
@@ -544,6 +544,11 @@ func (me *Metrics) RecordRequestQueueTime(success bool, requestType RequestType,
 
 }
 
+func (me *Metrics) RecordTimeoutNotice(sucess bool) {
+	// Not yet implemented for influx
```

Comment:
Why not?

Reply:
We don't use it, I don't know who does, and this is just for debugging facebook timeout notifications. Given Facebook's willingness to work with prebid, I doubt anyone else needs this, so wasn't worth the effort. Influx and Prometheus have never been 1:1 in metrics, due to their different feature sets.

Reply:
I understand that Xandr doesn't use Influx, but I'm quite sure there are others hosts who do use Influx and perhaps one of them will be successful in working with Facebook. I like to think we try to maintain parity between Influx and Prometheus.. although its admittedly not perfect.